File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
      int i=0;
if(nums.size()==0)
return 0;
for(int j=i+1;j<nums.size();)
{ if(nums[i]==nums[j])
j++;
else
{ i++;
nums[i]=nums[j];
}
}
return i+1;
        
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = len(nums) - 1
        slow = 0
        fast = 0
        while(fast <= l):
            if nums[slow] == nums[fast]:
                fast+=1
            elif nums[slow] != nums[fast]:
                slow+=1
                nums[slow] = nums[fast]
        return slow+1
        